notebooks/working/_04-tb-parking-choice-example.py

```diff
@@ -0,0 +1,101 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:light
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.4.0
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# # Purpose
+# This notebook will provide an illustration of a discrete choice model that tries to estimate causal effects, without making use of the framework outlined in Brathwaite and Walker (2018).
+# The study of note is
+# > Chiara, Giacomo Dalla, Lynette Cheah, Carlos Lima Azevedo, and Moshe E. Ben-Akiva. "A Policy-Sensitive Model of Parking Choice for Commercial Vehicles in Urban Areas." Transportation Science (2020).
+#
+# In this notebook we draw the causal diagram that we believes corresponds to the parking choice model in Chiara et al (2020).
+# We then point out some tests that could have been done and should be presented to justify this paper's causal model and its implicit assumptions, even before estimating parameters of the proposed choice models.
+# Finally, we will:
+# - note the importance of these implicit assumptions, and
+# - show how this study may present a real example of our simplified demsonstration of how incorrect causal graphs can lead to highly inaccurate causal effect estimates, even in selection-on-observables settings.
+#
+
+# +
+import sys
+
+sys.path.insert(0, '../../src/')
+from graphs import parking
+# -
+
+causal_graph = parking.PARKING_CAUSAL_MODEL.draw()
+causal_graph.graph_attr.update(size="10,6")
+causal_graph
+
```

Comment:
Should we add a description of all the variables in the graph?

Reply:
It feels like a lot of work (or maybe I'm just lazy). At the least, I should include a line in the module docstring that tells readers to check the paper for definitions of all variables.